network/controller.go

```diff
@@ -747,3 +748,27 @@ func (r *Controller) SetEventListener(l EventListener) {
 func (r *Controller) String() string {
 	return r.topo.String()
 }
+
+func getURLQuery(w rest.ResponseWriter, req *rest.Request) (l uint8, o uint8) {
```

Comment:
@nanpuhaha Please do not use short names (i.e., l and o) for a variable of primitive types. In this case, we know ```w``` and ```req``` are the ```rest.ResponseWriter``` and ```rest.Request```, respectively, but ```l``` and ```o``` are very confusing because they are primitive types.